lib/y2_installbase.pm

```diff
@@ -191,6 +192,23 @@ sub save_upload_y2logs {
     $self->investigate_yast2_failure() unless $args{skip_logs_investigation};
 }
 
+sub save_remote_upload_y2logs {
+    my ($self, %args) = @_;
+
+    return if (get_var('NOLOGS'));
+    $args{suffix} //= '';
+
+    type_string 'sed -i \'s/^tar \(.*$\)/tar --warning=no-file-changed -\1 || true/\' /usr/sbin/save_y2logs';
+    send_key 'ret';
+    my $filename = "/tmp/y2logs$args{suffix}.tar" . get_available_compression();
+    type_string "save_y2logs $filename\n";
+    my $uploadname = +(split('/', $filename))[2];
```

Comment:
is not the `+` operator only used with `print` or what is the purpose here?

Reply:
to be honest i found this online. perl magic trying to get the second element of the split

Reply:
it should work without the `+` like in [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/security/krb5/krb5_crypt_prepare.pm#L37)

Reply:
```
#! /usr/bin/perl -w
 
use strict;
use warnings;

my $filename = "/tmp/y2logs.targ.gz";
my $uploadname = (split('/', $filename))[2];
print "$uploadname\n";
```

Reply:
i am so insecure because i can say that in my runs it was failing. i am sure the one i use is working. i cant find the exact reason but i rather leave it as it is for now at least

Reply:
Unary "+" has no effect whatsoever, even on strings. It is useful syntactically for separating a function name from a parenthesized expression that would otherwise be interpreted as the complete list of function arguments. (See examples above under Terms and List Operators (Leftward).) from https://perldoc.perl.org/perlop.html#Symbolic-Unary-Operators

Reply:
ahm cool you searched :) but I think that does not apply this case, for example I found googling this use:
`my $module = reverse +( split /-/, reverse($string), 2 )[1]` and makes sense between the reverse and the rest of the line, but in your case there is not method calling that. Or I missing something?

Reply:
When i tested without the unary symbol it failed. So i went with what seems to work for sure.

Reply:
sure, not big deal.